cmd/semantic-release/main.go

```diff
@@ -57,7 +56,18 @@ func cliHandler(c *cli.Context) error {
 	ci := condition.NewCI()
 	logger.Printf("detected CI: %s\n", ci.Name())
 
-	repo, err := semrel.NewGithubRepository(context.TODO(), conf.GheHost, conf.Slug, conf.Token)
+	var (
+		repo semrel.Repository
+		err  error
+	)
+
+	switch ci.Name() {
+	case "Gitlab":
```

Comment:
Do we want a provider enum? Seems overkill for this single place

Reply:
This is fine for me 👍